pkg/bucket/interface.go

```diff
@@ -14,5 +14,7 @@ type Bucket interface {
 	Get(ctx context.Context, key string) (io.ReadCloser, error)
 
 	// List lists the matching object keys that have `prefix`.
+	// The prefix argument must end in /. (e.g. "foo/bar/").
```

Comment:
nit.
```suggestion
	// The prefix argument should end with /. (e.g. "foo/bar/").
```

Reply:
Thanks! I fixed it.
6f6bf886409b0f067572f066cf2aa0beb40086ef